src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -40,6 +40,32 @@ public Datastore() {
 		datastore = DatastoreServiceFactory.getDatastoreService();
 	}
 
+	/** Stores the User in Datastore. */
+	public void storeUser(User user) {
+		Entity userEntity = new Entity("User",user.getEmail());
```

Comment:
nit: use space after commas

Reply:
Can you also comment on why you initialized the Entity with ("User", user.getEmail())?